src/Controllers/CheckoutController.php

```diff
@@ -325,14 +359,16 @@ public function doKlarnaAuth(
     }
 
     /**
+     * @param int $orderId
      * @param PaymentService $paymentService
-     * @param  $orderId
      * @return string
+     * @throws \Throwable
      */
     public function doKlarnaAuthForReinit(
-        $orderId,
+        int $orderId,
```

Comment:
Are you sure that in no place NULL or something other than INT is given in here?

Reply:
Should be ok, order comes from 
 $eventDispatcher->listen('Ceres.LayoutContainer.MyAccount.OrderHistoryPaymentInformation', ReInitPaymentHook::class);
  $eventDispatcher->listen('Ceres.LayoutContainer.OrderConfirmation.AdditionalPaymentInformation', ReInitPaymentHook::class);